src/main/java/org/dataone/cn/indexer/object/ObjectManager.java

```diff
@@ -82,14 +65,10 @@ public class ObjectManager {
      */
     private ObjectManager() throws ServiceFailure, IllegalArgumentException, IOException {
         if (storage == null) {
-            if (storage == null) {
-                    storage = StorageFactory.getStorage();
-            }
+            storage = Storage.getInstance();
```

Comment:
(not critical): Would it be cleaner to move this up into the `static` block on line 51, so we're doing the static initialization all in one place?